### ISSUE TITLE
Changelog is out of date

### ISSUE THREAD

**Issue by chris-morgan** [kornelski/deunicode#10]
Last entry is 1.0.0.

While I’m at it, 1.2.1 and 1.3.1 are not tagged.

**Comment by svenstaro** [comment]
I agree. If the CHANGELOG isn't updated, it should be rather be deleted entirely. However, I still would prefer to have an up-to-date CHANGELOG.
